docs/api.md

```diff
@@ -22,6 +22,7 @@ Instead, it *returns* a new, connected component class, for you to use.
      - `force` *(Boolean)*: Forces the data to be always fetched when new props are received. Takes precedence over `comparison`.
      - `comparison` *(Any)*: Custom value for comparing this request and the previous request when the props change. If the `comparison` values are *not* strictly equal, the data will be fetched again. In general, it is preferred to rely on the default that compares material changes to the request (i.e. URL, headers, body, etc); however, this is helpful in cases where the request should or should not be fetched again based on some other value. If `force` is true, `comparison` is not considered.
      - `then(value, meta): request` *(Function)*: returns a request to fetch after fulfillment of this request and replaces this request. Takes the `value` and `meta` of this request as arguments. Return `undefined` in `then` to do side-effects after a successful request, leaving the request as is.
+      - `meta` is an object with keys `{ request, response, component, ...<your meta>}` where `component` is equal to the wrappedInstance. You can use it to create side effects on promise fulfillment. e.g. `then(value, meta) { meta.component.onDataLoaded(value); }`
```

Comment:
only if you passed `withRef: true` to options, otherwise `component` will be `undefined`